Makefile

```diff
@@ -1,5 +1,21 @@
 all:
 
+bf_replay_jit: bf_replay_jit.c
+	@if [ "${YK_DIR}" = "" ]; then \
+		echo "Please set YK_DIR in the environment"; \
+		exit 1; \
+		fi
```

Comment:
Over-indent?

Reply:
I don't think so, at least, not in make. FWIW vim's auto-indent agrees.

Reply:
vim's auto indent regularly does ridiculous things, so I'm not using that as a benchmark ;) If you remove the second tab, does the file not parse in `make`?

Reply:
It does work, but it I've never seen a make recipe indented in that way.

Reply:
OK OK OK, I take the hint :)